packages/asset-swapper/test/quote_requestor_test.ts

```diff
@@ -54,7 +55,7 @@ describe('QuoteRequestor', async () => {
                 endpoint: 'https://1337.0.0.1',
                 requestApiKey: apiKey,
                 requestParams: expectedParams,
-                responseData: successfulOrder1,
+                responseData: { signedOrder: successfulOrder1, quoteExpiry: successfulOrder1.expirationTimeSeconds },
```

Comment:
Nit but can we remove `quoteExpiry` from here so these are vanilla RFQT firm quotes?